core/src/main/java/org/apache/metamodel/query/builder/SatisfiedFromBuilder.java

```diff
@@ -44,6 +44,8 @@
 
     public FunctionSelectBuilder<?> select(FunctionType function, Column column, Object[] functionParameters);
 
+    public FunctionSelectBuilder<?> select(Column column, FunctionType function, Object[] functionParameters);
```

Comment:
This is the same as the one just above, but just with the arguments in a different order - is that really worth introducing?